README.md

```diff
@@ -14,7 +14,7 @@ Join our [Slack channel](https://slack.openmainframeproject.org/) to connect wit
 Client-side prerequisites for development:
 
 - Install [Node.js](https://nodejs.org/en/download/) v14.0 or later.
-- Install [Yarn](https://yarnpkg.com/getting-started/install).
+- Install [Yarn](https://yarnpkg.com/getting-started/install) v1.22.19.
```

Comment:
I'm wondering if we should update the Yarn link to point to the classic Yarn subdomain since we are still using v1? That way users do not have to click on the classic banner at the top to get to v1 instructions.

```suggestion
- Install [Yarn](https://classic.yarnpkg.com/en/docs/install) v1.22.19.
```

Reply:
Should we say to install Yarn Classic? I don't think the exact version number matters - if Yarn releases a 1.22.20 patch, we'd likely support it.

Reply:
Thanks all, changes pushed.  feel free to push in updates to my PRs for Docs updates in the future 😉